exercises/practice/saddle-points/.meta/example.jl

```diff
@@ -0,0 +1,8 @@
+function saddlepoints(matrix)
+    isempty(matrix) && return []
+
+    (cols_min,) = findmin(matrix, dims=1)
+    (rows_max,) = findmax(matrix, dims=2)
+
+    [Tuple(i) for i = CartesianIndices(matrix) if matrix[i] <= cols_min[i[2]] && matrix[i] >= rows_max[i[1]]]
```

Comment:
No idea if this is a reasonable implementation.

Reply:
It might be faster to not find the `rows_max` but only check if a column minimum is also a maximum in its row but I haven't tested it.

Reply:
I think the original way is better. Better worst case behaviour. If we write an approaches intro we can talk about how to do it differently.

Reply:
Okay cool! I'll revert